pack/parallel/rod_encounter.json

```diff
@@ -0,0 +1,77 @@
+        "code": "90004",
+        "double_sided": true,
+        "encounter_code": "read_or_die",
+        "encounter_position": 1,
+        "faction_code": "mythos",
+        "name": "Read or Die",
+        "pack_code": "rod",
+        "position": 4,
+        "quantity": 1,
+        "text": "<b>Easy / Standard</b>\n[skull]: -X. X is the number of [[Tome]] assets Daisy Walker controls.\n[cultist]: Reveal another token. If you fail, discard the top 2 cards of your deck.\n[tablet]: -2. If this is an attack or evasion attempt against Namer of the Dead and you do not succeed by at least 2, it attacks you.\n[elder_thing]: -3. If you fail, Daisy Walker takes 1 horror.",
+        "type_code": "scenario"
+    },
+    {
+        "back_flavor": "A tremendous whirlwind and a thunderous voice erupts from the Necronomicon, sending a tornado of pages into the sky. A sinister shadow looms over the entire campus, then escapes through the ensuing tempest. When it is done, the grimoire clatters to the floor, devoid of energy. Whatever presence it held before is now gone. It feels somehow empty in your hands.\n",
+        "back_name": "Reading Can Be Deadly",
+        "back_text": "<b>(→R2)</b>",
+        "code": "90005",
+        "doom": 12,
+        "double_sided": true,
+        "encounter_code": "read_or_die",
+        "encounter_position": 2,
+        "faction_code": "mythos",
+        "flavor": "Whatever entity has been unleashed from within the Necronomicon, it is spreading across the campus like a deadly shadow. You must put a stop to it before it grows too powerful to be contained.",
+        "illustrator": "Yoann Boissonnet",
+        "name": "Mortal Inquiry",
+        "pack_code": "rod",
+        "position": 5,
+        "quantity": 1,
+        "stage": 1,
+        "text": "Dr. Henry Armitage does not take up an ally slot and gains: \"You have 2 additional hand slots, which can only be used to hold [[Tome]] assets.\"\n\"Jazz\" Mulligan gains: \"[action]: Move to a [[Miskatonic]] location up to 3 connections away.\"",
+        "type_code": "agenda"
+    },
+    {
+        "back_flavor": "You stand in the restricted section of the Orne Library, surrounded by volumes of ancient, heretical texts. The incantations you have found may prove to be enough to disep the entity—or perhaps bind it back within the Necronomicon from whence it came. Your voice rises to a crescendo as you repeat the words. Shadows crawl across the room. The walls and ceiling bend. Pages flutter in the temptuous gale. Every fiber of your hair is whisked into the air. All is swept away in the ensuing chaos, like a wet paintbrush kissing the canvas of reality.",
```

Comment:
typo: 'disep' to 'dispel'

Reply:
Also, please add spaces around the dashes: 'entity - or' rather than 'entity-or'

Reply:
Typo: 'temptuous' to 'tempestuous'